src/shared/ml-thirdparty-client/config.ts

```diff
@@ -0,0 +1,34 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the Apache License, Version 2.0 (the 'License') and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Google
+ - Raman Mangla <ramanmangla@google.com>
+ --------------
+ ******/
+
+const config = {
+  PARTICIPANT_ID: 'pisp',
+  ALS_ENDPOINT: 'account-lookup-service:4002',
+  THIRDPARTY_REQUEST_ENDPOINT: 'thirdparty-api-adapter:3008',
+  TRANSACTION_REQUEST_ENDPOINT: 'transaction-request-service:4003',
+  PEER_ENDPOINT: '172.17.0.2:3001',
+}
+
+export default config
```

Comment:
Can you please update and move the config schema that is written in the `index.ts` file instead of creating a new one here? Besides, I believe that we shouldn't just hardcode the values like this because `ml-thirdparty-client` is intended to be a configurable plugin. It would make more sense to treat the current hardcoded values as default config instead.